CLI/src/CLI.cs

```diff
@@ -267,8 +267,9 @@ private void AddProgramsToRootTable(SymbolTable rootTable, IEnumerable<Program>
         {
             if (_configuration.ExecToStep >= ExecutionStep.SemanticCheck)
             {
-                foreach (var program in programs)
+                foreach (var program in programs.Where(p => p.IsMainProgram))
                 {
+                    // a stacked program should not be referenced from another source file
```

Comment:
Comment is not in-sync with the latest code since we also filter out nested programs.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1650/commits/d13214d60251eda9d164857a4179ed920b19dd0f